letters/modernized/14-letters.adoc

```diff
@@ -141,8 +148,10 @@ to whom my spirit has been nearly united.
 May the Lord have us all in his keeping to the end of time,
 is the desire of your affectionate friend,
 
+[.signed-section-signature]
 Catherine Phillips.
 
+[.letter-heading]
 Extracts From An Epistle Of Stephen Crisp.
```

Comment:
So there are a couple letters in here that are inserted that aren't numbered (this is the first one). They don't seem to belong with the letters surrounding them. Just wanted to bring that to your attention.